Automattic/wp-calypso
# Upload avatar guided tour

This PR adds a guided tour triggered from the onboarding checklist.

# Testing

* Visit `/checklist/<domain>` for your site
* Click 'Do It' for the user avatar task (whether completed or not - the tour should show)
* Verify that the following steps appear in the tour:

![user-avatar](https://user-images.githubusercontent.com/1926/34091328-b0cefdf2-e408-11e7-80ed-ffa732d39ce3.gif)

# Issues

* the tour does not trigger the first time the checklist item is clicked
* clicking 'All good, continue' on the first step before uploading an image will result in the tour immediately ending
